composer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "bigbank/php-xades-document",
+  "description": "XML Advanced Electronic Signatures Document",
+  "keywords": [
+    "ADOC",
+    "digital signing",
+    "bigbank"
+  ],
+  "license": "proprietary",
```

Comment:
Apache-2.0 is open source and less ambiguous than "proprietary"; however the decision to use it is "it depends" (on project goals).